App.tsx

```diff
@@ -1,12 +1,19 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, Text, View } from 'react-native';
-import { globalStyles } from 'style';
+
+import StatusButton from './src/components/StatusButton';
```

Comment:
używajmy aliasów: `@/components`

Reply:
używajmy też plików reeksportujących. Zgodnie ze strukturą w readme. Plik reeksportujący to plik `index.ts`, w readme jest to `index.tsx`, ale zostanie to poprawione.

Oprócz tego możesz wrzucić ten komponent w folder `/components/Button`